src/scripts/main.js

```diff
@@ -5,4 +5,42 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const mainTab = document.querySelector('.dashboard');
+
+for (let i = 0; i < people.length; i++) {
```

Comment:
Try not to use for loop but rather forEach, map, reduce and filter, here forEach would be the most fitting
```js
people.forEach(person => { ... })
```